docs/source/examples/merge_dataset.rst

```diff
@@ -0,0 +1,69 @@
+################
+ Merge Datasets
+################
+
+This topic describes the merge dataset operation:
+
+Take the `Oxford-IIIT Pet <https://gas.graviti.cn/dataset/data-decorators/OxfordIIITPet>`_
+and `Dogs vs Cats <https://gas.graviti.cn/dataset/data-decorators/DogsVsCats>`_
+as examples. Their structures looks like::
+
+    Oxford-IIIT Pet/
+        test/
+            Abyssinian_002.jpg
+            ...
+        trainval/
+            Abyssinian_001.jpg
+            ...
+
+    Dogs vs Cats/
+        test/
+            1.jpg
+            10.jpg
+            ...
+        train/
+            cat.0.jpg
+            cat.1.jpg
+            ...
+
+There are lots of pictures of cats and dogs in these two datasets,
+merge them to get a more diverse dataset
+
+.. note::
+
+   Before merging datasets operation, fork them first.
```

Comment:
Would this be more formal?
`Before merging datasets, fork both of the open datasets first.`